ecology-letters.csl

```diff
@@ -73,9 +73,15 @@
     <choose>
       <if type="webpage">
         <group>
-          <text value="URL" suffix=" "/>
-          <text variable="URL"/>
+          <text value="Available at:" suffix=" "/>
```

Comment:
Could you change this to 
```<text term="available at" text-case="capitalize-first" suffix=": "/>```
I'm trying to avoid using ```text value``` as much as possible.